pages/common/nextclade.md

```diff
@@ -0,0 +1,28 @@
+# nextclade
+
+> Bioinformatics tool for virus genome alignment, clade assignment and qc checks.
+> More information: <https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html>.
+
+- Run Nextclade to create TSV report, auto-downloading the latest [d]ataset:
+
+`nextclade run -d {{dataset-name}} {{path/to/fasta}} -t {{path/to/output-tsv}}`
+
+- Show all available datasets:
+
+`nextclade dataset list`
+
+- Download latest SARS-CoV-2 dataset:
+
+`nextclade dataset get --name sars-cov-2 --output-dir {{path/to/output-dir}}`
```

Comment:
```suggestion
`nextclade dataset get --name sars-cov-2 --output-dir {{path/to/directory}}`
```

We can use a generic placeholder for `--output-dir` argument because option has long and descriptive name.